### PR TITLE
Add support for bundled transactions (FIP-11b)

### PR DESCRIPTION
**Pull request by 0xCasey**

### CODE REVIEW

File: src/transactions/signed/AddBundledTransactions.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        this.validationData = { fioAddress, tpid: technologyProviderId || null }
+        this.validationRules = validationRules.registerFioAddress
+    }
+
+    public getData(): any {
+        const actor = this.getActor()
+        const data = {
+            fio_address: this.fioAddress,
+            bundleSets,
```
Comment (andreyvEze) [inline src/transactions/signed/AddBundledTransactions.ts:29]: here should be `bundle_sets: this.bundleSets,`

File: src/transactions/signed/AddBundledTransactions.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { SignedTransaction } from './SignedTransaction'
+import { Constants } from '../../utils/constants'
+import { validationRules } from '../../utils/validation'
+
+export class AddBundledTransactions extends SignedTransaction {
+    public ENDPOINT: string = 'chain/burn_fio_address'
```
Comment (andreyvEze) [inline src/transactions/signed/AddBundledTransactions.ts]: is that right api endpoint? `burn_fio_address`

File: src/transactions/signed/AddBundledTransactions.ts (hunk 1 above)
Comment (andreyvEze) [reply src/transactions/signed/AddBundledTransactions.ts:29]: but I don't know what key should be - `bundle_sets` or something else

File: tests/index.spec.js [hunk 3]
```diff
@@ -408,6 +408,28 @@ describe('Testing generic actions', () => {
...
+
+    expect(result).to.have.all.keys('fee')
+    expect(result.fee).to.be.a('number')
+  })
+
+  it(`add Bundled Transactions`, async () => {
+    const result = await fioSdk.genericAction('addBundledTransactions', {
+      fioAddress: newFioAddress,
+      bundleSets: bundleSets,
+      newOwnerKey: pubKeyForTransfer,
```
Comment (andreyvEze) [inline tests/index.spec.js]: the `bundleSets` is undefined, we need to set some value above, for example `const bundleSets = 3`
also there is no `newOwnerKey` prop for that method, I think we should remove this line

File: tests/testnet.spec.js [hunk 4]
```diff
@@ -366,6 +366,28 @@ describe('Testing generic actions', () => {
...
+
+    expect(result).to.have.all.keys('fee')
+    expect(result.fee).to.be.a('number')
+  })
+
+  it(`add Bundled Transactions`, async () => {
+    const result = await fioSdk.genericAction('addBundledTransactions', {
+      fioAddress: newFioAddress,
+      bundleSets: bundleSets,
+      newOwnerKey: pubKeyForTransfer,
```
Comment (andreyvEze) [inline tests/testnet.spec.js]: the same as for index.test

**Review by andreyvEze — CHANGES_REQUESTED** [review]
check comments

**Comment by 0xCasey** [comment]
> check comments

Changes have been applied. Thanks for the set of eyes on this.

**Review by andreyvEze — APPROVED** [review]

**Pull request merged**